x/ibctransfer/whitelisting.go

```diff
@@ -45,8 +46,97 @@ func OnRecvPacketWhiteListed(
 			Events: ctx.EventManager().Events().ToABCIEvents(),
 		}, acknowledgement.GetBytes(), nil
 	}
+	// get result of transfer receive
+	recvResult, resBytes, err := sdkAppModule.OnRecvPacket(ctx, packet)
+	// if no error and packet is returning and needs conversion: convert
+	if err != nil && IsRecvPacketReturning(packet, data) && checkRecvConvert(ctx, whitelistKeeper, packet, data) {
```

Comment:
sdkAppModule.OnRecvPacket doesn't ever return an error, it converts it to an error acknowledgement packet in the returned bytes.

Will need to copy the module.OnRecvPacket code here instead of calling it, so that we can do the call to the keeper.OnRecvPacket() to get the error.